### PR TITLE
LF: release LF 1.13

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
CHANGELOG_BEGIN

 - LF: Add support for BigNumeric.

 - Ledger API: bump version for LF1.13 

CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: daml-lf/language/daml-lf.bzl [hunk 1]
```diff
@@ -10,8 +10,7 @@
 lf_version_configuration = {
     "legacy": "1.8",
     "stable": "1.12",
-    "latest": "1.12",
-    "preview": "1.13",
+    "latest": "1.13",
```
Comment (remyhaemmerle-da) [inline daml-lf/language/daml-lf.bzl:13]: Not really sure of what I am doing here.

File: ledger/ledger-api-test-tool-on-canton/BUILD.bazel [hunk 2]
```diff
@@ -63,14 +63,16 @@ conformance_test(
     ],
     lf_versions = [
         "stable",
-        "latest",
+        # FIXME: add latest once canton-test-runner supports LF 1.13
+        #  "latest",
```
Comment (remyhaemmerle-da) [inline ledger/ledger-api-test-tool-on-canton/BUILD.bazel:67]: @oliverse-da the `FIXME` is for you

File: ledger/ledger-api-test-tool-on-canton/BUILD.bazel [hunk 3]
```diff
@@ -103,7 +105,8 @@ conformance_test(
     ],
     lf_versions = [
         "stable",
-        "latest",
+        # FIXME: add latest once canton-test-runner supports LF 1.13
```
Comment (remyhaemmerle-da) [inline ledger/ledger-api-test-tool-on-canton/BUILD.bazel:108]: @oliverse-da this one as well

File: daml-lf/language/daml-lf.bzl (hunk 1 above)
Comment (cocreature) [reply daml-lf/language/daml-lf.bzl:13]: I think `stable` is the compiler default and `latest` is stable but not default? We really need some comments in this file and perhaps also more sensible names.

File: daml-lf/spec/daml-lf-1.rst [hunk 4]
```diff
@@ -235,8 +235,8 @@ Version: 1.12
     details
 
 
-Version: 1.13 (preview)
-.......................
+Version: 1.13
```
Comment (cocreature) [inline daml-lf/spec/daml-lf-1.rst:238]: You might want to copy the changelog from 1.dev. `A BigNumeric` is a bit brief and also seems like a typo.

File: daml-lf/spec/transaction.rst [hunk 5]
```diff
@@ -181,7 +181,7 @@ later.
 +--------------------+-----------------+
 |                 12 |      2021-01-27 |
 +--------------------+-----------------+
-|       (preview) 13 |      2021-04-06 |
+|                 13 |      2021-04-06 |
```
Comment (cocreature) [inline daml-lf/spec/transaction.rst:184]: do we track the dates when we make it a preview or when we make it stable?

File: docs/source/support/compatibility.rst [hunk 6]
```diff
@@ -43,6 +43,8 @@ The below lists with which Daml Connect version a new Ledger API version was int
 
    * - Ledger API Version
      - Daml Connect Version
+   * - 1.11
```
Comment (cocreature) [inline docs/source/support/compatibility.rst:46]: Please also add an entry below in the list of changes.

**Review by cocreature — APPROVED** [review]
Thanks!

File: daml-lf/spec/transaction.rst (hunk 5 above)
Comment (remyhaemmerle-da) [reply daml-lf/spec/transaction.rst:184]: I do not know. :man_shrugging:
Dates in the specs are a mess.

Comment (remyhaemmerle-da) [reply daml-lf/spec/transaction.rst:184]: I try to not change the date once introduced.

File: docs/source/support/compatibility.rst (hunk 6 above)
Comment (remyhaemmerle-da) [reply docs/source/support/compatibility.rst:46]: This change makes sens ? I was really not sure of what I was doing here.

Comment (cocreature) [reply docs/source/support/compatibility.rst:46]: This change is fine. We introduce Ledger API version 1.11 in SDK 1.14. What is missing is an entry below that explains the changes in Ledger API version 1.11

Comment (remyhaemmerle-da) [reply docs/source/support/compatibility.rst:46]: Does this change make sens ? I really not sure of what I am doing here.

Comment (remyhaemmerle-da) [reply docs/source/support/compatibility.rst:46]: :+1:

**Pull request merged**